relation_processor.py

```diff
@@ -0,0 +1,181 @@
+# Process data and create dataloader for models
+# For binary relation extraction
+
+import pandas as pd
+from collections import Counter
+from sklearn.model_selection import train_test_split
+from torch import LongTensor
+from torch.nn.utils.rnn import pad_sequence
+from torch.utils.data import TensorDataset, DataLoader
+
+
+class RelationProcessor:
```

Comment:
For a future PR, or in this one if you want, we should create two seperate classes

1. Class only does the preprocessing of the data
2. The other class does the splititng of the dataset/creating dataloader (or maybe even that is 3 classes)

This is good for now tho